static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
So the example you see above, and [its usage below](https://github.com/guardian/frontend/pull/9062/files#diff-8f39a2d435918d56716f6b8525622049L23), is a dynamic require? Surely this should have been routed through `System.normalize` then?